.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+1.
+1.
```

Comment:
uber nit: should the list be 1. 2. 3. ? I know we approved/merged the UI template with the 1. 1. 1. sequence but I never asked why in that PR 🙃

Reply:
The 1. 1. 1. sequence just creates an ordered list that goes from 123. for example:

1.
1.
1.

Reply:
Ah I see, so GH markdown renders 1.1.1. properly to a 123 list , I thought 1.2.3 is *required* in the template to have the 123 render. Thanks, works for me in any case!

Reply:
https://daringfireball.net/projects/markdown/syntax#list Explains this quite well